### PR TITLE
modify scripts and commands that run zeek

### PR DESCRIPTION
**Pull request by alfred-landrum**
This is the brimsec/zeek repo side of https://github.com/brimsec/brim/issues/731.

The script (mac/linux) or executable (windows) that runs Zeek, including creating required environment variables & command line options, is now `zeekrunner` or `zeekrunner.exe` . The command line options used to run Zeek are now here in this repo instead of in the zqd source. The job object based process termination for windows is removed, as it is moved into zqd; this will make it easier to eventually replace the windows zeek runner with a script or batch file.

### CODE REVIEW

File: .github/workflows/brim-release.yml [hunk 1]
```diff
@@ -2,8 +2,8 @@ name: Brim release
 
 on:
   push:
-    tags:
-      - v*brim*
+    branches:
+      - add-zeek-args
```
Comment (alfred-landrum) [inline .github/workflows/brim-release.yml:6]: I'll revert the changes to this file before merging.

**Comment by alfred-landrum** [comment]
Ready for review: I've got PR's open in zq & brim that show pcap ingest works, using the artifact from this PR (including making a manual artifact for windows using this code).

File: brim/release [hunk 2]
```diff
@@ -1,4 +1,6 @@
-#!/bin/sh -ex
+#!/usr/bin/env bash
+
+set -xeo pipefail
```
Comment (nwt) [inline brim/release:3]: Is this necessary?

Comment (alfred-landrum) [reply brim/release]: no, reverted

**Pull request merged**